src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -1989,6 +1989,18 @@
   "modifier.tconstruct.fireprimer": "火种",
   "modifier.tconstruct.fireprimer.flavor": "煽风点火！",
   "modifier.tconstruct.fireprimer.description": "增加工具生成火焰的面积",
+  "modifier.tconstruct.shears": "Shearing",
+  "modifier.tconstruct.shears.flavor": "Snip snip!",
+  "modifier.tconstruct.shears.description": "Tool can be used to shear mobs and disarm tripwire",
+  "modifier.tconstruct.silky_shears": "Silky Shears",
+  "modifier.tconstruct.silky_shears.flavor": "Snip snip!",
+  "modifier.tconstruct.silky_shears.description": "Tool can shear mobs if the silky modifier is added",
+  "modifier.tconstruct.silky_aoe_shears": "Silky Shears",
+  "modifier.tconstruct.silky_aoe_shears.flavor": "Snip snip!",
+  "modifier.tconstruct.silky_aoe_shears.description": "Tool can shear mobs if the silky modifier is added",
+  "modifier.tconstruct.harvest": "Harvest",
+  "modifier.tconstruct.harvest.flavor": "Swish!",
+  "modifier.tconstruct.harvest.description": "Tool can be used on crops to harvest and replant them",
```

```suggestion
  "modifier.tconstruct.shears": "修剪",
  "modifier.tconstruct.shears.flavor": "咔嚓咔嚓！",
  "modifier.tconstruct.shears.description": "工具可用于修剪生物和剪除绊线",
  "modifier.tconstruct.silky_shears": "丝剪",
  "modifier.tconstruct.silky_shears.flavor": "咔嚓咔嚓！",
  "modifier.tconstruct.silky_shears.description": "若加持有“丝触”强化，则工具可用于修剪生物",
  "modifier.tconstruct.silky_aoe_shears": "丝剪",
  "modifier.tconstruct.silky_aoe_shears.flavor": "咔嚓咔嚓！",
  "modifier.tconstruct.silky_aoe_shears.description": "若加持有“丝触”强化，则工具可用于修剪生物",
  "modifier.tconstruct.harvest": "收割",
  "modifier.tconstruct.harvest.flavor": "春耕秋获！",
  "modifier.tconstruct.harvest.description": "工具可收割并重新种植农作物",
```
Translating `Swish!` with a single interjection seems quite off to me.
This version of translation of that key means literally "sow/plough in spring, harvest in autumn"

---

pretty good, maybe we should discuss the `Swish!`
看起来好极了! 我们可能得讨论下`Swish!` 的中文翻译